docs/source/api/plugin/landing-pages.mdx

```diff
@@ -187,6 +210,118 @@ If you set this option to `false`, your server's landing page is a splash page c
 </tbody>
 </table>
 
+### `embed` options
+
+These are the fields you can include in the `embed` option you pass to the `ApolloServerPluginLandingPageLocalDefault`:
+
+<table class="field-table api-ref">
+  <thead>
+    <tr>
+      <th>Name /<br/>Type</th>
+      <th>Description</th>
+    </tr>
+  </thead>
+
+<tbody>
+
+<tr>
+<td>
+
+##### `initialState`
+
+`Object`
+
+</td>
+<td>
+
+An object containing additional display options related to the visual state of the embedded Explorer on page load.
+
+For supported subfields, see [`initialState` options](#embedinitialstate-options).
+
+</td>
+</tr>
+
+<tr>
+<td>
+
+##### `endpointIsEditable`
+
+`boolean`
+
+</td>
+<td>
+
+By default, the embedded Sandbox has a URL input box that is editable by users.
+
+Set `endpointIsEditable` to `false` to prevent users of your Apollo Server instance from changing the endpoint URL.
+</td>
+</tr>
+
+
+</tbody>
+</table>
+
+
+### `initialState` options
+
+These are the fields you can include in the `initialState` option you pass to `embed` under `ApolloServerPluginLandingPageLocalDefault`:
+
+<table class="field-table api-ref">
+  <thead>
+    <tr>
+      <th>Name /<br/>Type</th>
+      <th>Description</th>
+    </tr>
+  </thead>
+
+<tbody>
+<tr>
+<td>
+
+
+###### `pollForSchemaUpdates`
```

Comment:
I have no idea which one is right but we have a solid mix of 5# and 6# throughout the docs (and both seem to style fine, but it does give us a mix of h5/h6 in the dom). Just noting, I'll address it in a different PR.

Reply:
@StephenBarlow says h6!